components/engine_rocks/src/rocks_metrics.rs

```diff
@@ -1115,11 +1115,13 @@ pub fn flush_engine_properties(engine: &DB, name: &str) {
 // For property metrics
 #[rustfmt::skip]
 lazy_static! {
-    pub static ref STORE_ENGINE_SIZE_GAUGE_VEC: IntGaugeVec = register_int_gauge_vec!(
+    pub static ref STORE_ENGINE_SIZE_HISTOGRAM_VEC: HistogramVec = register_histogram_vec!(
         "tikv_engine_size_bytes",
         "Sizes of each column families",
-        &["db", "type"]
+        &["db", "type"],
+        exponential_buckets(1048576.0, 2.0, 20).unwrap()
```

Comment:
So if there are 1024 DBs, and the base size will be 1048576 * 1024 * 3 = 3G?

Reply:
I don't understand the calculation. Why is histogram bucket size related to sample number?

Reply:
Never mind, I thought a tablet was a db.